src/main/resources/openapi.json

```diff
@@ -1100,6 +1100,24 @@
                 "operationId": "openapi",
                 "description": "Retrieves the OpenAPI v2 specification in JSON format."
             }
+        },
+        "/": {
+            "get": {
+                "responses": {
+                    "200": {
+                        "description": "Info about Kafka Bridge instance configuration.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "string"
```

Comment:
it should be now a JSON with more information not just a string. For now, it's a JSON with just bridge version field. Can we define a new type definition? Furthermore, we should have an example.